.evergreen.yml

```diff
@@ -230,6 +239,24 @@ functions:
           nvm install --no-progress --default ${node_version}
           npm install -g yarn
 
+  sym-link:
+    command: shell.exec
+    params:
+      working_dir: parsley
+      shell: bash
+      script: |
+        ln -s evergreen/graphql/schema sdlschema
+
+  yarn-codegen:
+    command: shell.exec
+    params:
+      working_dir: parsley
+      shell: bash
+      script: |
+        ${PREPARE_SHELL}
+        chmod +x ./scripts/check_codegen.sh
```

Comment:
If we ensure the file is considered executable by git, is this necessary? (more info [here](https://stackoverflow.com/questions/40978921/how-to-add-chmod-permissions-to-file-in-git))

Reply:
gave it a try, but maybe something about Evergreen makes it not work 🤔